images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -179,18 +183,21 @@ function Start-DownloadWithRetry
 
     $filePath = Join-Path -Path $DownloadPath -ChildPath $Name
 
-    #Default retry logic for the package.
+    # Default retry logic for the package.
     while ($Retries -gt 0)
     {
         try
         {
+            $downloadStartTime = Get-Date
             Write-Host "Downloading package from: $Url to path $filePath ."
             (New-Object System.Net.WebClient).DownloadFile($Url, $filePath)
             break
         }
         catch
         {
+            $failTime = [math]::Round(($(Get-Date) - $downloadStartTime).TotalSeconds, 2)
             Write-Host "There is an error during package downloading:`n $_"
+            Write-Host "Total time elapsed $($failTime)"
```

Comment:
It's not a total time, it's time to a single attempt.

Reply:
Agree. Missed this point.